packages/plugin-ext/src/plugin/notebook/notebooks.ts

```diff
@@ -312,6 +312,26 @@ export class NotebooksExtImpl implements NotebooksExt {
         return result;
     }
 
+    waitForNotebookDocument(uri: TheiaURI, duration = 2000): Promise<NotebookDocument> {
```

Comment:
should this timeout maybe be configurable somewhere?

Reply:
I don't think that's necessary. The race condition is usually in a fairly small time frame, on local host it's roughly ~10ms. Having a larger timeout should rarely be an issue, while 2000ms is probably enough for most if not all use cases.